src/pykeen/models/base.py

```diff
@@ -495,10 +496,10 @@ def predict_scores_all_tails(
         """
         # Enforce evaluation mode
         self.eval()
-        if slice_size is None:
-            scores = self.score_t(hr_batch)
+        if slice_size is not None and self.can_slice_t:
```

Comment:
With the recent changes, every model should allow slicing for h/r/t.

Reply:
Also, `slice_size` is not part of the `score_*` API, at least for `Interaction`, so we shouldn't need these if-else parts here.